proto/viam/app/v1/app.proto

```diff
@@ -153,9 +156,13 @@ service AppService {
   // Deletes an IdentityAuthorization
   rpc RemoveRole(RemoveRoleRequest) returns (RemoveRoleResponse);
 
-  // Shows organization, location, and robot level permissions that exist on the resource
+  // Returns all authorization roles for a given resource
   rpc ListAuthorizations(ListAuthorizationsRequest) returns (ListAuthorizationsResponse);
 
+  // Returns a user's authorization roles in the org
+  rpc ListAuthorizationsForUser(ListAuthorizationsForUserRequest) returns (ListAuthorizationsForUserResponse);
```

Comment:
one thing to note when writing this - you only want to return the resources in each identity authorization that contains a resource that the current app user also has access to. We don't want to return info about resources that other users have access to that the current user doesn't have access to / shouldn't even know exist.